contracts/src/multisig.cairo

```diff
@@ -1,21 +1,17 @@
 use array::ArrayTrait;
 use option::OptionTrait;
 
-fn assert_unique_values<T,
-impl TCopy: Copy<T>,
-impl TDrop: Drop<T>,
-impl TPartialEq: PartialEq<T>,
+fn assert_unique_values<
+    T, impl TCopy: Copy<T>, impl TDrop: Drop<T>, impl TPartialEq: PartialEq<T>, 
 >(
     a: @Array::<T>
 ) {
     let len = a.len();
     _assert_unique_values_loop(a, len, 0_usize, 1_usize);
 }
 
-fn _assert_unique_values_loop<T,
-impl TCopy: Copy<T>,
-impl TDrop: Drop<T>,
-impl TPartialEq: PartialEq<T>,
+fn _assert_unique_values_loop<
+    T, impl TCopy: Copy<T>, impl TDrop: Drop<T>, impl TPartialEq: PartialEq<T>, 
```

Comment:
I think you need to run `cairo-format src/**/*.cairo`, the linting fails here otherwise

Reply:
my local cairo-format doesnt seem to pick this up 🤔 
```
local setup:
cairo-lang-formatter 1.0.0-rc0
cairo: 1.0.0-rc0
scarb 0.2.0-alpha.1
```

I'll fix this manually for now